packages/block-library/src/cover/style.scss

```diff
@@ -84,7 +84,13 @@
 	}
 
 	&.components-placeholder {
-		height: inherit;
```

Comment:
Sorry I missed this, but I feel all this selector `&.components-placeholder` should not be in this file but it should be moved to `editor.scss` as it's editor specific.

Reply:
Hi @youknowriad, I also noticed this problem during my review, the styles that should be part of editor.scss were not only this placeholder, so preferred to merge the change right away and address the other problem in a separate PR https://github.com/WordPress/gutenberg/pull/12236.